libs/site-semantic-theme/internal/webpack.less.js

```diff
@@ -5,6 +5,11 @@
 //
 const { paths, modifyVars } = require('./vars.js');
 
+// In dev mode, leave the modules classnames intact so we can debug easier
+const modules = process.env.NODE_ENV !== 'production'
+  ? { localIdentName: "[name]__[local]--[hash:base64:5]" }
+  : true;
```

Comment:
un-hash style names - debugging was painful without this

Reply:
You don't have to tell me

Reply:
"Move fast and break things" is a terrible motto